src/Tribe/Tickets_Handler.php

```diff
@@ -1063,15 +1052,15 @@ public function save_tickets_order( $post_id ) {
 
 		// If our data is missing or we're already in the middle of saving, bail
 		if (
-			empty( $_POST[ 'tribe_tickets_order' ] ) ||
-			! empty( $this->saving_order ) ||
-			! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) )
+			empty( $_POST[ 'tribe_tickets_order' ] )
+			|| ! (
+				isset( $_POST[ 'tribe-tickets-post-settings' ] )
```

Comment:
That format was requested by @borkweb here: https://github.com/moderntribe/event-tickets/pull/418/files/e409517d814b8ccb2aeb85b82f22cce1e0761788#diff-ed1e9dc57f0161d5dd32c4d53c1a7096